toDo.js

```diff
@@ -0,0 +1,36 @@
+// document listener waits for all data to load
+document.addEventListener("DOMContentLoaded" ,()=>{
+
+  let submit = document.querySelector(".submit")
+  //declare the submit button
+  let toDo = document.querySelector(".todo")
+  //declare the toDo list so we can interact with it
+  let thing = document.querySelector(".thing")
+  //declare the items that will be li's for the toDo list
+  let body = document.querySelector("body")
+  //declare the body
+  let pTag = document.querySelector("p")
+
+  submit.addEventListener("click",()=>{
+  //when the submit button is clicked we want to run our function
+    let newThing =document.createElement("li");
+    function addThing(){
+      //our function will evaluate the input from the items section
+      newThing.innerText = thing.value;
+      toDo.appendChild(newThing)
+    }
+
+    let warningP =document.createElement("p");
+    function warning(){
+      warningP.innerText = "You didn't insert an item. Please insert a new item";
+      body.append(warningP)
+    }
+    if(thing.value.length>0){addThing()}
+    else{warning()}
+    })
+
+    toDo.addEventListener("click",()=>{
+
+      event.target.innerHTML= "<strike>" + event.target.innerText + "</strike>"
+    })
+  })
```

Comment:
Nice job with this assignment! Some notes:

- Love when I see comments in code!
- Your code is allowing strings composed of empty spaces to be added to the list, which isn't against the instructions, but you could use the [String.prototype.trim() method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) to check for these types of input
- Currently, your code is adding a new error message to the body every time that an invalid input is entered. You can fix this by creating your `warningP` variable and appending it to the body _outside_ of your `submit` event listener, then just changing the innerText of `warningP` if the input is invalid inside of your warning function

Let me know if you've got any questions!